digitalfabrik/entitlementcard
# 1664: Untouched State

### Short description

As a user i don't want to get initial error message if i did not interact with the form

### Proposed changes



- add untouched states for self service form
- add sendAttempt state and pass them to extensions to also show errors if a send attempts was made without filling required fields

### Side effects



- none

### Testing

1. Open /erstellen
2. No error messages should be shown initially to the user
3. Click "Koblenz Pass erstellen" without filling anything or not filling all required fields. All fields should be validated and show error messages
4. Test if all fields still will be validated correctly



### Resolved issues



Fixes: #1664
